docs/content/configuration/coordinator.md

```diff
@@ -22,6 +22,7 @@ The coordinator node uses several of the global configs in [Configuration](../co
 
 |Property|Description|Default|
 |--------|-----------|-------|
+|`druid.coordinator.beOverlord`|Boolean value for whether this coordinator node should act like an overlord as well. This configuration allows users to simplify a druid cluster by not having to deploy any standalone overlord nodes. If set to true, then make sure `druid.selectors.indexing.serviceName` is correctly set to coordinator service name everywhere and all the overlord configuration must be added to the coordinator node configuration.|true|
```

Comment:
The need to change `druid.selectors.indexing.serviceName` means migration can't be done totally smoothly:

- If you set beOverlord on coordinators first, one of them might grab leadership and now other servers can't find the overlord service (since they would look for the old serviceName and the new leader won't announce on that one).
- If you update druid.selectors.indexing.serviceName on other servers first, then they won't be able to find the current leader if it's still an overlord.

This could be fixed by letting the coordinator announce the overlord service name for a while too, just so stuff can find it. Then the order for migration would be: first update coordinators to beOverlord and announce both services, then update other Druid servers to set druid.selectors.indexing.serviceName equal to the coordinator name, then (optionally) update coordinators to stop announcing the old overlord service name.